instances.tf

```diff
@@ -0,0 +1,74 @@
+variable instance_type {
+  default = "t2.nano"
+}
+
+data "aws_ami" "linux_connectivity_tester" {
+  most_recent = true
+
+  filter {
+    name = "name"
+
+    values = [
+      "connectivity-tester-linux*",
+    ]
+  }
+
+  owners = [
+    "093401982388",
+  ]
+}
+
+resource "aws_instance" "BastionHostLinux" {
+  ami                    = "${data.aws_ami.linux_connectivity_tester.id}"
+  instance_type          = "${var.instance_type}"
+  subnet_id              = "${aws_subnet.OPSSubnet.id}"
+  vpc_security_group_ids = ["${aws_security_group.Bastions.id}"]
+
+  tags {
+    Name = "${local.name_prefix}ec2-linux"
```

Comment:
Additional tags required, as per dq routing doc:
computed values should result in the below:

```hcl
  tags {
    Name             = "bastionlinux01-dq-bastions-dq-ops-prod"
    Service          = "dq-bastions"
    EnvironmentGroup = "dq-ops"
    Environment      = "prod"
  }
```

Reply:
in hindsight I think we should revisit naming and tagging post ITHC. will update the board accordingly